packages/app/src/components/swap/ConfirmSwapModal.tsx

```diff
@@ -118,18 +130,25 @@ const ConfirmSwapModal: FC<Props> = ({
                 {`Confirm Swap ${tokenSendedSymbol} for ${tokenReceivedSymbol}`}
               </Heading>
             </CardHeader>
-            <CardBody justify="center" align="center">
-              {confirmStatus === "None" && (
-                <>
-                  <Form>
-                    <Box justify="around" align="center">
+            <CardBody>
+              <Box pad={{ top: "medium", horizontal: "large" }} align="center">
+                {confirmStatus === "None" && (
+                  <Form style={{ width: "100%" }}>
+                    <Box gap="medium">
                       <FormField label={`Send ${tokenSendedSymbol}`}>
                         <TextInput
                           readOnly
                           id="formSwapSend"
-                          type="number"
+                          type="text"
```

Comment:
if you did that to disable the up and down arrows in firefox, there's a better way of doing it:
https://stackoverflow.com/questions/23372903/hide-up-down-arrow-buttons-spinner-in-input-number-firefox-29
I think we are already doing thix in index.css but with a webkit prefix, maybe that's the issue

Reply:
We are using text input fields everywhere else. Would be good to stick to one for consistency. The actual type the state begins with is also a string.

Reply:
it's not a question of type but what the form allows to input. you cannot type text to numeric inputs - it's a string nonetheless